CWE/events/italy.txt

```diff
@@ -1,24 +1,59 @@
-#Italy becomes a republic
+#Abdication of King Vittorio Emmanuele III
 country_event = {
-  id = 8007704
-  title = EVT_8007704_NAME
-  desc = EVT_8007704_DESC
-  picture = "nwo2_italian_constitutional_referendum"
-  fire_only_once = yes
-
+	id = 8007757
+	title = "The Abdication of King Victor Emmanuel III"
+	desc= "EVTDESC7"
+	picture = "pavorotti"
+	
+	fire_only_once = yes
+	
 	trigger = {
 		tag = ITA
-		year = 1946 NOT = { year = 1950 }
 		government = hms_government
-		war = no
-		is_vassal = no
+		NOT = { year = 1950 }
```

Comment:
Please follow the style convention.

trigger = {
TAG
YEAR

GOV TYPE
is_vassal = yes/no
war = yes/no

ruling_party_ideology (if necessary)

country flags (if necessary)
}

Reply:
Missed the TAG whoops, but I realized that  without specifying the year id it's 1946 it fires more on time, while if I specify year = 1946 it maye fire inbetween may and january of the year after, tho if necessary will fix.

Reply:
I feel it is better to specify a date to make the code clearer for third parties who may want to edit these events in the future.